posthog/settings/web.py

```diff
@@ -212,6 +212,13 @@
     "EXCEPTION_HANDLER": "exceptions_hog.exception_handler",
     "TEST_REQUEST_DEFAULT_FORMAT": "json",
     "DEFAULT_SCHEMA_CLASS": "drf_spectacular.openapi.AutoSchema",
+    # These rate limits are applied to all Django views.
+    # Note: Ingestion + decide endpoints do not use Django views, so no rate limits are applied
+    "DEFAULT_THROTTLE_CLASSES": [
+        "posthog.rate_limit.PassThroughBurstRateThrottle",
+        "posthog.rate_limit.PassThroughSustainedRateThrottle",
+    ],
+    "DEFAULT_THROTTLE_RATES": {"burst": "120/minute", "sustained": "1000/hour"},
```

Comment:
Do these make sense outside of cloud? If not, we should put this behind an env var.

Reply:
Good call, just put it behind a `RATE_LIMIT_ENABLED` env var